tests/VersionRouteTest.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
+namespace RonasIT\Support\Tests;
+
+use Illuminate\Support\Facades\Route;
+use RonasIT\Support\Contracts\VersionEnumContract;
+use RonasIT\Support\Enum\VersionEnum;
+use RonasIT\Support\Routing\RouteFacade;
+
+class VersionRouteTest extends HelpersTestCase
+{
+    protected const ROUTE_FACADE = '/test-facade';
+    protected const ROUTE_OBJECT = '/test-object';
+
+    protected bool $init = false;
+
+    public function setUp(): void
+    {
+        parent::setUp();
+
+        $this->app->bind(VersionEnumContract::class, VersionEnum::class);
+
+        if (!$this->init) {
+            $versionFrom = $this->createMock(VersionEnumContract::class);
+            $versionFrom->value = VersionEnum::v1;
+            $versionTo = $this->createMock(VersionEnumContract::class);
+            $versionTo->value = VersionEnum::v2;
+            Route::group(['prefix' => 'v{version}'], function () use ($versionTo, $versionFrom) {
+                Route::get(static::ROUTE_OBJECT, function () {
+                    return 'Route';
+                })->versionRange($versionFrom, $versionTo);
+            });
```

Comment:
let's move it to the separate methods of the new trait `RouteMockTrait`

Reply:
Done.